compat-data-schema.md

```diff
@@ -1,334 +1,306 @@
 
+#### Browser identifiers
 
-#### `"prefix"`
-Contains the prefix to add to the sub-feature name (default to the empty
-string). Note that leading and trailing `-` must be included.
+The currently accepted browser identifiers are:
+* `webview_android`, Webview, the former stock browser on Android,
+* `chrome`, Google Chrome (on desktops),
+* `chrome_android`, Google Chrome (on Android),
+* `edge`, MS Edge (on Windows),
+* `edge_mobile`, MS Edge, the mobile version,
+* `firefox`, Mozilla Firefox (on desktops),
+* `firefox_android`, Firefox for Android, sometimes nicknamed Fennec,
+* `ie_mobile`, Microsoft Internet Explorer, the mobile version,
+* `ie`, Microsoft Internet Explorer (discontinued),
+* `nodejs` Node.js JavaScript runtime built on Chrome's V8 JavaScript engine,
+* `opera`, the Opera browser (desktop), based on Blink since Opera 15,
+* `opera_android`, the Opera browser (Android version),
+* `safari`, Apple Safari, on Mac OS,
+* `safari_ios`, Apple Safari, on iOS.
+
+No browser identifier is mandatory.
+
+#### The `support_statement` object
+This object is the key element for each browser.
+It is an array of `simple_support_statement` objects, but if there
+is only one of them, the array can be omitted.
+
+
+Example of an `array_support_statement` compat object (with 2 entries):
+```json
+"support": {
+  "firefox": [
+    {
+      "version_added": "6.0"
+    },
+    {
+      "prefix": "-moz-",
+      "version_added": "3.5",
+      "version_removed": "9.0"
+    }
+  ]
+}
+```
+
+Example of a `simple_support_statement` compat object (with 1 entry, array omitted):
+```json
+"support": {
+  "ie": { "version_added": "6.0" }
+}
+```
+
+### Compat data in `simple_support_statement` objects
+This is the core object containing the compatibility information for a browser.
+It consist of the following properties:
+
+#### `version_added`
+This is the only mandatory property and it contains a string with the version
+number indicating when a sub-feature ha been added (and is therefore supported).
+The Boolean values indicate that a sub-feature is supported (`true`, with the
+additional meaning that it is unknown in which version support was added) or
+not supported (`false`). A value of `null` indicates that support information is
+entirely unknown. Examples:
+
+* Support from version 3.5 (inclusive):
+```json
+{
+ "version_added": "3.5"
+}
+```
+* Supported, but version unknown:
+```json
+{
+  "version_added": true
+}
+```
+* No support:
+```json
+{
+  "version_added": false
+}
+```
+* Support unknown (default value, if browser omitted):
+```json
+{
+  "version_added" : null
+}
+```
+
+#### `version_removed`
+Contains a string with the version number the sub-feature is
+removed in. It may also be a Boolean value of (`true` or `false`), or the
+`null` value. If `version_added` is set to a Boolean or a `string`, `version_removed`
+default value is `false`; if it is `null`, the default value of `version_removed` is `null` too. Examples:
```

Comment:
"if it is" -> "it" here is ambiguous and this sentence is hard to parse. Maybe:

* If `version_added` is set to `true`, `false`, or a string, `version_removed` defaults to `false`
* if`version_added` is set to `null`, the default value of `version_removed` is also `null`.